content/guidelines/reviewing.en.md

```diff
@@ -0,0 +1,109 @@
+The pull request created will consist of fewer checks than those that add declarations. The checks will guide the reviewer on the key things to look out for during the review process, and a link to inspect the declaration.
+
+For PRs that update declarations, you should focus should be on two things: history file and the declaration.
+
+- **History file:** The history file is a JSON file that keeps track of a service declaration changes. It contains a `validUntil` property that the date a specific version of a service declaration was last tracked. You have to confirm that this date is the same as the date in the issue opened for the declaration when the bot couldn't track it. This issue is usually included in the PR message. The history file is updated with every `update` PR. You can find more information about the history file [here](https://docs.opentermsarchive.org/contributing-terms/#maintaining-declarations).
+- **Declaration:** for `update` PRs, you only look at the selectors to make sure they are **simple** and also verifiying the **generated declaration** version is relevant and ok.
+
+### Step-by-step Review Guide
+
+1. Click on the inspect the declaration suggestion link to view contribution using the contribution tool.
+2. Check both the significant and insignificant parts of the document.
+    - Ensure that only [simple/basic selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_selectors/Selectors_and_combinators#basic_selectors) are used and avoid complex selectors like [combinators](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_selectors/Selectors_and_combinators#combinators) and [pseudo selectors](https://www.w3schools.com/css/css_pseudo_elements.asp). To simplify, try to avoid selectors with indexes/numbers e.g `:nth-child(n)` and long chains e.g. `div > body > .aside > p > span > a )`
+3. Verify the version of the document in the contribution tool by clicking on the `verify version` button.
+4. Open the issue linked with the pull request. Confirm the date when the declaration was last tracked from the bot's comment.
+5. Compare it with the `validUntil` property in the history file under the `Files changes` section of the PR. If the dates are the same, proceed to add approve.
+6. Merge the contribution.
+
+You can read more about maintaining declarations from the [official documentation](https://docs.opentermsarchive.org/contributing-terms/#maintaining-declarations).
+
+## Making Changes to a Contribution
+
+When you spot an error in a contribution, instead of asking them to implement these changes, we usually recommend you fix it, especially for first time contributors,in order to save time. Make your corrections directly through the contribution tool, and send the document. Doing this, will create an update in the already existing pull request and a new comment will be generated by the OTA-bot.
```

Comment:
```suggestion
When you spot an error in a contribution, instead of asking the author to implement these changes, we usually recommend you fix it, especially for first time contributors, in order to speed up the process. You can make your corrections directly through the graphical user interface and send the document. This will create an update in the already existing pull request and a new comment will be generated by the OTA-bot.
```